js/framework/AnalyticsConstants.js

```diff
@@ -39,7 +39,7 @@ if (!OO.Analytics.STREAM_TYPE)
 
 /**
  * @public
- * @description These are the Ooyala Player error codes
+ * @description (NOTE: will deprecate) These are the Ooyala Player error codes
```

Comment:
You can just write [DEPRECATED].  Doesn't mean it totally doesn't work but we aren't supporting it anymore.